src/psyclone/tests/domain/lfric/kernel_interface_test.py

```diff
@@ -39,36 +39,23 @@
 
 '''
 
-# pylint: disable=no-name-in-module
+# p#y#l#int: disable=no-name-in-module
```

Comment:
??

Reply:
I'm having problems with pylint in wsl2 but I think the pylint disable is not needed so I've removed it.